fimsauthz-api/fims-authz.yaml

```diff
@@ -101,14 +103,38 @@ paths:
                   type: string
```

Comment:
Let's add yet another note.  We COULD theoretically reject requests that re-use salt.  Not sure when/if we'd implement this feature, but can state:  "NOTE3: If a salt value is used in multiple requests, FIMS AZ may reject duplicated salt requests beyond the first."